mobile-widgets/qml/SsrfSwitch.qml

```diff
@@ -18,8 +18,8 @@ Switch {
 		Rectangle {
 			x: root.checked ? parent.width - width : 0
 			y: parent.height / 2 - height / 2
-			width: Kirigami.Units.largeSpacing * 1.1
-			height: Kirigami.Units.largeSpacing * 1.1
+			width: 20 * PrefDisplay.mobile_scale
+			height: 20 * PrefDisplay.mobile_scale
```

Comment:
not disagreeing with the change, but what I find odd is that line 9 to line 21 used to have about 2x difference, vs. it now is a 1.8 difference.
I assume there's a good reason for that and it now looks better? :-)